web/src/beta/components/SettingsButtons/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
```

Comment:
Don't need to import React. Since we use vite's react plugin vite will inject React in to all components on build 👍🏻